examples/heat-source-mpi.py

```diff
@@ -172,6 +173,9 @@ def rhs(t, u):
                         ], overwrite=True)
 
         u = rk4_step(u, t, dt, compiled_rhs)
+        # Force evaluation once per timestep
+        u = thaw(freeze(u), actx)
```

Comment:
Why not use _force_evaluation here?

Reply:
How's https://github.com/illinois-ceesd/mirgecom/pull/638/commits/19f50233371b10a94cb971dfee203f9c0fa6e1cf?

Reply:
LGTM, thanks!